### DISCUSSION TITLE
Distibute Infection as PHAR

### DISCUSSION THREAD

**Discussion by martinssipenko** [infection/infection#1694] [Ideas]
Hi,

Have you thought about distributing Infection as PAHR, similar to what PHPStan does? That is when installing PHPStan via composer the PHAR is installed instead of the source code which removes issues with conflicting package versions.

**Comment by maks-rafalko** [comment]
Yes, we thought about it.

Related issue: https://github.com/infection/infection/issues/956
Also related: https://github.com/infection/infection/pull/1686#issuecomment-1116945560 - here I explain that at least `Mutator` and `Node` (from PHPParser) won't be prefixed, but the same thing is in PHPStan's PHAR distribution.

if further discussion is needed, let's do it on the issue.
